--- centos-release-7/x86_64/CentOS-Base.repo	2020-04-05 20:50:43.000000000 -0400
+++ centos-release-7.9-ciq/x86_64/CentOS-Base.repo	2024-04-28 23:52:02.311884397 -0400
@@ -12,7 +12,8 @@
 
 [base]
 name=CentOS-$releasever - Base
-mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
+baseurl=https://ciq:$ciq_access_token@repository.ciq.com/v1/mirror/centos-7.9/centos-7.9-mirror/7.9.2009/os/$basearch/
 #baseurl=http://mirror.centos.org/centos/$releasever/os/$basearch/
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
@@ -20,7 +21,8 @@
 #released updates 
 [updates]
 name=CentOS-$releasever - Updates
-mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=updates&infra=$infra
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=updates&infra=$infra
+baseurl=https://ciq:$ciq_access_token@repository.ciq.com/v1/mirror/centos-7.9/centos-7.9-mirror/7.9.2009/updates/$basearch/
 #baseurl=http://mirror.centos.org/centos/$releasever/updates/$basearch/
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
@@ -28,7 +30,8 @@
 #additional packages that may be useful
 [extras]
 name=CentOS-$releasever - Extras
-mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras&infra=$infra
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras&infra=$infra
+baseurl=https://ciq:$ciq_access_token@repository.ciq.com/v1/mirror/centos-7.9/centos-7.9-mirror/7.9.2009/extras/$basearch/
 #baseurl=http://mirror.centos.org/centos/$releasever/extras/$basearch/
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
@@ -36,7 +39,8 @@
 #additional packages that extend functionality of existing packages
 [centosplus]
 name=CentOS-$releasever - Plus
-mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=centosplus&infra=$infra
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=centosplus&infra=$infra
+baseurl=https://ciq:$ciq_access_token@repository.ciq.com/v1/mirror/centos-7.9/centos-7.9-mirror/7.9.2009/centosplus/$basearch/
 #baseurl=http://mirror.centos.org/centos/$releasever/centosplus/$basearch/
 gpgcheck=1
 enabled=0
--- centos-release-7/x86_64/CentOS-CR.repo	2020-04-05 20:50:43.000000000 -0400
+++ centos-release-7.9-ciq/x86_64/CentOS-CR.repo	2024-04-28 23:52:02.317884344 -0400
@@ -22,7 +22,8 @@
 
 [cr]
 name=CentOS-$releasever - cr
-baseurl=http://mirror.centos.org/centos/$releasever/cr/$basearch/
+baseurl=https://ciq:$ciq_access_token@repository.ciq.com/v1/mirror/centos-7.9/centos-7.9-mirror/7.9.2009/cr/$basearch/
+#baseurl=http://mirror.centos.org/centos/$releasever/cr/$basearch/
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
 enabled=0
--- centos-release-7/x86_64/CentOS-Debuginfo.repo	2020-04-05 20:50:43.000000000 -0400
+++ centos-release-7.9-ciq/x86_64/CentOS-Debuginfo.repo	2024-04-28 23:52:02.322884300 -0400
@@ -14,7 +14,8 @@
 
 [base-debuginfo]
 name=CentOS-7 - Debuginfo
-baseurl=http://debuginfo.centos.org/7/$basearch/
+baseurl=https://ciq:$ciq_access_token@repository.ciq.com/v1/mirror/centos-7.9/centos-7.9-debuginfo-mirror/7/$basearch/
+#baseurl=http://debuginfo.centos.org/7/$basearch/
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Debug-7
 enabled=0
--- centos-release-7/x86_64/CentOS-fasttrack.repo	2020-04-05 20:50:43.000000000 -0400
+++ centos-release-7.9-ciq/x86_64/CentOS-fasttrack.repo	2024-04-28 23:52:02.327884256 -0400
@@ -2,7 +2,8 @@
 
 [fasttrack]
 name=CentOS-7 - fasttrack
-mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=fasttrack&infra=$infra
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=fasttrack&infra=$infra
+baseurl=https://ciq:$ciq_access_token@repository.ciq.com/v1/mirror/centos-7.9/centos-7.9-mirror/7.9.2009/fasttrack/$basearch/
 #baseurl=http://mirror.centos.org/centos/$releasever/fasttrack/$basearch/
 gpgcheck=1
 enabled=0
