diff --color -uNr centos-release-7/x86_64/CentOS-Base.repo centos-release-7.mod/x86_64/CentOS-Base.repo
--- centos-release-7/x86_64/CentOS-Base.repo	2020-04-05 20:50:43.000000000 -0400
+++ centos-release-7.mod/x86_64/CentOS-Base.repo	2024-07-02 18:23:32.105946080 -0400
@@ -12,25 +12,31 @@
 
 [base]
 name=CentOS-$releasever - Base
-mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os&infra=$infra
 #baseurl=http://mirror.centos.org/centos/$releasever/os/$basearch/
+baseurl=https://dl.rockylinux.org/vault/centos/7.9.2009/os/$basearch/
 gpgcheck=1
+enabled=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
 
 #released updates 
 [updates]
 name=CentOS-$releasever - Updates
-mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=updates&infra=$infra
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=updates&infra=$infra
 #baseurl=http://mirror.centos.org/centos/$releasever/updates/$basearch/
+baseurl=https://dl.rockylinux.org/vault/centos/7.9.2009/updates/$basearch/
 gpgcheck=1
+enabled=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
 
 #additional packages that may be useful
 [extras]
 name=CentOS-$releasever - Extras
-mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras&infra=$infra
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras&infra=$infra
 #baseurl=http://mirror.centos.org/centos/$releasever/extras/$basearch/
+baseurl=https://dl.rockylinux.org/vault/centos/7.9.2009/extras/$basearch/
 gpgcheck=1
+enabled=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
 
 #additional packages that extend functionality of existing packages
